example/README.md

```diff
@@ -1,73 +1,43 @@
-# plaid-react-native-demo-app
+# Example App
 
-Sample apps using the React Native Plaid Link SDK on both iOS and Android.
+Our example app provides a minimal React Native app that implements Plaid Link. This app allows you to link a sample bank account.
 
-## Local Developement of react-native-plaid-link-sdk
+In order to test Plaid Link using this example app you'll need the following:
 
-To test local changes to the `react-native-plaid-link-sdk` package...
+- A [Plaid Account](https://dashboard.plaid.com/signup) to get a API keys.
+- A [Link Token](https://plaid.com/docs/api/tokens/#linktokencreate) you can quickly fetch one using your API keys and our [Postman collection](https://plaid.com/docs/api/postman/).
 
-* Run `tsc` from the project's root (`npm install --global typescript` if command not found) to compile the typescript source. Ensure the `dist` directory has been created
-* Run `cd example`
-* Run `npm install` to resolve example app dependencies
-* Copy the local package implementation into `example/node_modules`:<br> From `example` run `npm run sync-local-pkg`
+## Running the example app
 
-Run the `npm run sync-local-package` each time you make changes to the local package, then reload the react native app.
+> Note: This setup assumes you've already setup your React Native [development environment](https://reactnative.dev/docs/environment-setup). 
 
-At this point you should be able to build & run the app using the local package as a dependency.
+From the root directory of our SDK. 
 
-Make sure you do not check these changes in! To get out of local dev mode, revert the 
-changes to `example/package.json` & run `npm install` again. 
+1. `cd example`
+2. `npm install`
+3. `cd ios`
+4. `bundle install`
+5. `bundle exec pod install`
+6. `npx react-native start`
+7. `npx react-native run-ios` or `npx react-native run-android`
 
-### iOS
+[Running on simulator](https://reactnative.dev/docs/running-on-simulator-ios)
+[Running on emulator](https://stackoverflow.com/a/63994477/7245977)
```

Comment:
I think you probably intended to have a linebreak between these but it isn't showing up

Reply:
Fixed